CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.36.0
+------------------------------
+*November 29, 2022*
+
+### Changed
```

Comment:
does this need to be marked as a breaking change if anywhere isn't currently calling the utility mixin?